js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/AutomaterializeLeftPanel.tsx

```diff
@@ -50,48 +60,82 @@ export const AutomaterializeLeftPanel = ({
 };
 
 interface ListProps {
-  assetHasDefinedPartitions: boolean;
-  evaluations: AutoMaterializeEvaluationRecordItemFragment[];
-  onSelectEvaluation: (evaluation: AutoMaterializeEvaluationRecordItemFragment) => void;
-  selectedEvaluation?: AutoMaterializeEvaluationRecordItemFragment;
+  definition?: AssetViewDefinitionNodeFragment | null;
+  evaluations: AssetConditionEvaluationRecordFragment[];
+  onSelectEvaluation: (evaluation: AssetConditionEvaluationRecordFragment) => void;
+  selectedEvaluation?: AssetConditionEvaluationRecordFragment;
 }
 
 export const AutomaterializeLeftList = (props: ListProps) => {
-  const {assetHasDefinedPartitions, evaluations, onSelectEvaluation, selectedEvaluation} = props;
+  const {evaluations, onSelectEvaluation, selectedEvaluation, definition} = props;
 
   return (
-    <Box
-      padding={{vertical: 8, horizontal: 12}}
-      style={{flex: 1, minHeight: 0, overflowY: 'auto'}}
-      flex={{grow: 1, direction: 'column'}}
-    >
-      {evaluations.map((evaluation) => {
-        const isSelected = selectedEvaluation?.evaluationId === evaluation.evaluationId;
-        const {numRequested, numSkipped, numDiscarded} = evaluation;
-
-        return (
-          <EvaluationListItem
-            key={`skip-${evaluation.timestamp}`}
-            onClick={() => {
-              onSelectEvaluation(evaluation);
-            }}
-            $selected={isSelected}
-          >
-            <Box flex={{direction: 'column', gap: 4}}>
-              <TimestampDisplay timestamp={evaluation.timestamp} />
-              <EvaluationCounts
-                numRequested={numRequested}
-                numSkipped={numSkipped}
-                numDiscarded={numDiscarded}
-                isPartitionedAsset={assetHasDefinedPartitions}
-                selected={isSelected}
-              />
+    <Box flex={{grow: 1, direction: 'column'}}>
+      <Box padding={{vertical: 12, horizontal: 24}} border="bottom">
+        <Subtitle1>Evaluations</Subtitle1>
+      </Box>
+      <Box
+        padding={{bottom: 8, horizontal: 12}}
+        style={{flex: 1, minHeight: 0, overflowY: 'auto'}}
+        flex={{grow: 1, direction: 'column'}}
+      >
+        <Box border="bottom" padding={{top: 8, bottom: 12, left: 12, right: 8}}>
+          <Link to="/overview/automaterialze">
```

Comment:
This doesn't apply to the sensor case though, right?

Reply:
ah yeah